lib/libutee/include/tee_api_defines_extensions.h

```diff
@@ -7,6 +7,10 @@
 #ifndef TEE_API_DEFINES_EXTENSIONS_H
 #define TEE_API_DEFINES_EXTENSIONS_H
 
+/* Algorithm Identifiers */
```

Comment:
Please replace this comment with something more meaningful such as `/* RSA signatures with MD5 hash */`